docs/LV_client/tutorials/Weather.md

```diff
@@ -0,0 +1,7 @@
+## Change Weather
+
+Learn how to change the weather using the LabVIEW Client. The monoDrive simulator allows you to simulate clear days, sunny days, sunsets, rain, and fog. 
```

Comment:
Learn how to change the weather using the LabVIEW Client. The monoDrive simulator allows you to simulate the any of the following weather conditions during the day or at sunset.: sunny, cloudy, rain, fog, etc.